runtime-modules/membership/src/lib.rs

```diff
@@ -489,6 +536,149 @@ decl_module! {
 
             Self::deposit_event(RawEvent::ReferralCutUpdated(value));
         }
+
+        /// Transfers invites from one member to another.
+        #[weight = 10_000_000] // TODO: adjust weight
+        pub fn transfer_invites(
+            origin,
+            source_member_id: T::MemberId,
+            target_member_id: T::MemberId,
+            number_of_invites: u32
+        ) {
+            Self::ensure_member_controller_account_signed(origin, &source_member_id)?;
+
+            let mut source_membership = Self::ensure_membership(source_member_id)?;
```

Comment:
Lets avoid having these juicy mutables flying around in here, ony tears and misery will result :D.

Reply:
Agreed and fixed.